WooCommerce/Classes/ViewRelated/Products/Variations/Edit Attributes/AttributeOptionListSelectorCommand.swift

```diff
@@ -52,7 +52,7 @@ final class AttributeOptionListSelectorCommand: ListSelectorCommand {
 private extension AttributeOptionListSelectorCommand {
     enum Localization {
         static let anyOption = NSLocalizedString(
-            "Any Attribute",
+            "Any %@",
```

Comment:
Here we should use [positional placeholders](https://github.com/woocommerce/woocommerce-ios/blob/develop/docs/localization.md#always-use-positional-placeholders)!

Reply:
Thanks for catching that! I'll update that and another place where that string is used, to avoid having two different strings for translation.